features/google/src/components/importExplanationExpandable/importExplanationExpandable.jsx

```diff
@@ -0,0 +1,329 @@
+import React, { useContext, useState } from "react";
+// import InfoBox from "../infoBox/infoBox.jsx";
+import {
+    PolyButton,
+    PolyImportContext,
+    RoutingWrapper,
+} from "@polypoly-eu/poly-look";
+import { GoogleContext } from "../../context/google-context.jsx";
+import { useHistory } from "react-router-dom";
+import { FileSelectionError, FileImportError } from "@polypoly-eu/poly-import";
+// import ScrollButton from "../buttons/scrollButton/scrollButton.jsx";
+// import scrollSmoothly from "../../utils/smoothScroll.js";
+import "./importExplanationExpandable.css";
+
+const isSectionOpened = (section, importStatus, importSteps) => {
+    return {
+        request:
+            section == importStatus || importStatus == importSteps.beginning,
+        download: section == importStatus,
+        import: section == importStatus,
+        explore: section == importStatus,
+    }[section];
+};
+
+const ImportExplanationExpandable = ({
+    importSteps,
+    importSections,
+    importStatus,
+    onUpdateImportStatus,
+}) => {
+    const { pod, runWithLoadingScreen, setGlobalError } =
+        useContext(GoogleContext);
+    const { files, refreshFiles, handleRemoveFile } =
+        useContext(PolyImportContext);
+
+    const importIds = {
+        request: "request",
+        download: "download",
+        import: "import",
+        explore: "explore",
+    };
+
+    const history = useHistory();
+
+    // const expandableRef = useRef();
+    const expandableId = "expandable";
+
+    // useEffect(() => {
+    //     scrollSmoothly(importIds[importStatus], expandableId, ["progress-bar"]);
+    // }, [importStatus]);
+
+    const handleRequestStatus = () => {
+        onUpdateImportStatus(importSteps.download);
+        window.pod.polyNav.openUrl("https://www.facebook.com/dyi");
+    };
+
+    const handleExampleDataRequest = () => {
+        onUpdateImportStatus(importSteps.import);
+        window.pod.polyNav.openUrl("example-data-download");
+    };
+
+    const handleDownloadDataLinkClick = () => {
+        onUpdateImportStatus(importSteps.import);
+        window.pod.polyNav.openUrl("https://www.facebook.com/dyi");
+    };
+
+    const formatSize = (size) => {
```

Comment:
Looks like this is copied from the facebook feature, up to and including URLs. For the time being, change URLs. Possibly for the future, factor out these common functions, possibly all of them, but `formatSize` is probably a low-hanging fruit.

Reply:
We've addressed this one yesterday, the links are now all Google specific. We'll need to push refactoring out (more) common behaviour for later :/